x/twap/types/errors.go

```diff
@@ -62,3 +62,14 @@ type InvalidRecordCountError struct {
 func (e InvalidRecordCountError) Error() string {
 	return fmt.Sprintf("The number of records do not match, expected: %d\n got: %d", e.Expected, e.Actual)
 }
+
+type InvalidRecordTimeError struct {
```

Comment:
I'm wondering if we should change this to `InvalidUpdateRecordError` or something similar, since this error does not only happen when it has a invalid record **time** but it also includes invlaid record height as well. Lmk what you think